code/game/objects/items/storage/pouch.dm

```diff
@@ -444,6 +444,28 @@
 	storage_slots = 5
 	max_storage_space = 10
 	can_hold = list(/obj/item/reagent_containers/syringe)
+	
+/obj/item/storage/pouch/hypospray
+	name = "hypospray pouch"
+	desc = "It can contain hyposprays and autoinjectors."
+	icon_state = "syringe"
+	storage_slots = 3 //2 less than injectors yet more flexible options
+	can_hold = list(/obj/item/reagent_containers/hypospray)
+
+
+/obj/item/storage/pouch/hypospray/corps
+	name = "Corps hypospray pouch"
+	desc = "It can contain hyposprays and autoinjectors, this one got a terragov corpsmans logo on its back."
```

Comment:
Punctuation and capitalization issues. Better wording might be "It can contain hyposprays and autoinjectors, this one has a Terragov corpsman logo on its back."